### PR TITLE
disaster

### PR DESCRIPTION
**Pull request by stubbEnable**

### CODE REVIEW

**Comment by Volas171** [comment]
disaster h

**Comment by DarkBrick** [comment]
hm

File: src/main/java/mindustry/plugin/commands/ModeratorCommands.java [hunk 1]
```diff
@@ -245,6 +246,36 @@ public void registerCommands(CommandHandler handler){
...
+                PlayerData pd = Database.getData(args[0]);
+                PlayerInfo info;
+                if(args[0].length() == 24) { // uuid length
+                    info = netServer.admins.getInfo(args[0]);
+                }else{
+                    ctx.sendEmbed(false, ":mag: can't find that player/uuid..");
+                    return;
+                }
+                if (pd != null && info != null) {
+                    Database.updateData(args[0], pd);
```
Comment (fuzzbuck) [inline src/main/java/mindustry/plugin/commands/ModeratorCommands.java:261]: why update it? you didnt change anything in PlayerData

File: src/main/java/mindustry/plugin/commands/ModeratorCommands.java [hunk 2]
```diff
@@ -245,6 +246,36 @@ public void registerCommands(CommandHandler handler){
             }
         });
 
+        //im sorry for this disaster, but we need it
+        handler.<Context>register("admin", "<uuid>", "Unadmin/Admin chosen player", (args, ctx) ->{
+            CompletableFuture.runAsync(() -> {
+                PlayerData pd = Database.getData(args[0]);
```
Comment (fuzzbuck) [inline src/main/java/mindustry/plugin/commands/ModeratorCommands.java:252]: is this required?

**Review by fuzzbuck — CHANGES_REQUESTED** [review]
PlayerData here is not required at all imo, remove it and use PlayerInfo purely

**Pull request merged**